### DISCUSSION TITLE
Can sourcemap uploading in Expo local development be disabled?

### DISCUSSION THREAD

**Discussion by bglgwyng** [getsentry/sentry-react-native#3264] [Q&A]
I am currently utilizing the Expo development build for my project. Whenever I make changes to native modules, I initiate the app build process using the command npx expo run:ios. However, I have encountered an issue with this approach - the command triggers the uploading of sourcemaps, which can be somewhat inconvenient.

In this context, I find it necessary to provide the SENTRY_AUTH_TOKEN environment variable while executing the build command. Yet, it seems redundant to upload the sourcemap for an app that is solely being tested locally. Is there a way to configure sentry-react-native to function only under specific conditions, such as the production or staging environment?

**Comment by krystofwoldrich** [comment]
Hi, 
thank you for the message,
you can use `SENTRY_ALLOW_FAILURE` env with this the error will be logged out but the build will continue.

You can read more about it here: https://docs.sentry.io/platforms/react-native/troubleshooting/#failing-build-due-to-sentry-cli

**Reply by bglgwyng** [reply]
Thanks!

**Reply by bglgwyng** [reply]
It worked! By the way, it would be better to disable this functionality explicitly, rather than just ignoring failure. 
sentry-expo is responsible for overseeing that particular feature.
But again, thanks!

**Reply by krystofwoldrich** [reply]
Yeah, we agree and added the flag to `@sentry/react-native` https://github.com/getsentry/sentry-react-native/releases/tag/5.11.0

**Reply by bglgwyng** [reply]
Thanks! Nice work!
